hana/lib/drivers/dynatrace.js

```diff
@@ -0,0 +1,91 @@
+// COPIED AS IS (excluding unused code) FROM @sap/cds
```

Comment:
Is this a long term or temporary solution ? As native support from `@sap/hana-client` was requested.

Reply:
Sebastian mentioned that Daniel agreed to implement that on our side as observability colleagues claimed the feature degradation to the legacy hana service.

I'm afraid that this will stay on us for a longer time.